tests/unit/model/test_datastore.py

```diff
@@ -265,3 +268,18 @@ def test_bump_datastore_major():
         'KJOENN': 'KJOENN__0_1.parquet',
         'SIVSTAND': 'SIVSTAND__0_1.parquet'
     }
+    assert len(_get_file_list_from_dir(Path(DATASTORE_ARCHIVE_DIR))) > 0
```

Comment:
> Would be nice if this expected 1
And then this expects 2 to make sure the previous archived file is not deleted

Reply:
Yessir!